spec/yaml/HotelSearch_v3_swagger_specification.yaml

```diff
@@ -368,11 +368,11 @@ definitions:
         format: date
       roomQuantity:
         description: number of rooms (1-9)
-        example: 1
-        type: integer
+        example: "1"
+        type: string
         minimum: 1
```

Comment:
I see the json is updated but not the yaml for this line. I recommend you to do a conversion of the .json file instead of modifying the .yaml as well to avoid forgetting a field.

Reply:
convert is done!